utils/cart-functions.js

```diff
@@ -17,12 +17,12 @@ export const getFloatVal = ( string ) => {
  * @param {Object} product Product
  * @param {Integer} productPrice Product Price
  * @param {Integer} qty Quantity
- * @return {{image: *, productId: *, totalPrice: number, price: *, qty: *, name: *}}
+ * @return {{image: *, databaseId: *, totalPrice: number, price: *, qty: *, name: *}}
  */
 export const createNewProduct = ( product, productPrice, qty ) => {
 
 	return  {
-		productId: product.productId,
+		databaseId: product.databaseId,
```

Comment:
We could have just updated this to `productId: product?.databaseId`, so we didn't have to replace it at other places, because databaseId is actually the product id.
But this is also fine.

Reply:
Sure, thanks!